client/scripts/uploader.js

```diff
@@ -1,7 +1,17 @@
+var is_correct_dimensions = function(url) {
+	var img = new Image()
```

Comment:
`var`

Reply:
same as above, var is the only supported thing to use